gateway/api/serializers.py

```diff
@@ -208,6 +211,22 @@ class RunJobSerializer(serializers.ModelSerializer):
     class Meta:
         model = Job
 
+    def is_trial(self, function: Program, author) -> bool:
```

Comment:
Serializers logic will need a big refactor but it's too much for the purpose of this PR so I just left it by now here.